source/Calamari/Deployment/PackageRetention/Repositories/JsonJournalRepository.cs

```diff
@@ -16,16 +17,24 @@ public class JsonJournalRepository
     {
         Dictionary<PackageIdentity, JournalEntry> journalEntries;
         const string SemaphoreName = "Octopus.Calamari.PackageJournal";
+        internal const string DefaultJournalName = "PackageRetentionJournal.json";
```

Comment:
Does this need to be exposed outside this class as `internal`? Can we make this `private` like the other variables?

Reply:
Rider says it can be made private but I use it in the test class which needs it to be exposed 🤔 Otherwise we'd have to duplicate the journal name in the test class.

Reply:
See my point above with the tests.